corehq/apps/callcenter/sync_user_case.py

```diff
@@ -57,18 +69,17 @@ def update_user_case(self, case, fields, close):
             close=close,
             update=fields
         )
```

Comment:
```suggestion
        ))
```
*** Error compiling 'corehq/apps/callcenter/sync_user_case.py'...
  File "corehq/apps/callcenter/sync_user_case.py", line 72
    self._user_case_changed(fields)
       ^
SyntaxError: invalid syntax